### PR TITLE
add rewards panels and change styles

### PR DESCRIPTION
**Pull request by kkzzhizhou**
1. stat show in the end of plugins
2. add rewards panels
3. fix the list padding problem
4. other panel small adjust

### CODE REVIEW

**Comment by SuperChamp234** [comment]
First, I want to start by saying I really appreciate your help in building the plugin! 
We're a bit new to this stuff so the code-review might take a bit time.

There seems to be no `RewardView` file [here](https://github.com/SuperChamp234/habitica-sync/blob/bc97bcfc3f12770b077db825d9defeec8ff6f2e0/src/view/Components/Taskview/index.tsx#L5) so the plugin won't build at all, might want to check that.

Our plugin doesn't really run in the background, so we don't really think a log is necessary [here](https://github.com/SuperChamp234/habitica-sync/blob/bc97bcfc3f12770b077db825d9defeec8ff6f2e0/src/main.ts#L18).

I'll leave more comments for questions I have, looking forward to your replies!

File: src/view.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class HabiticaSyncView extends ItemView {
         )
     }
 
-    async onClose(){
-        ReactDOM.unmountComponentAtNode(this.containerEl.children[1]);
-    }
+    // async onClose(){
```
Comment (SuperChamp234) [inline src/view.tsx:35]: This is basic template as to maintain views, can I ask why you've commented them?

Comment (kkzzhizhou) [reply src/view.tsx:35]: thanks for you reply.
- habitica  has the panel called "Rewards",so i add this panel in this plugins.hope this plugins to fully function.but may be have some humanized way to closed it as wish.
- the console.log is a mistake when i pull the request. by the way, i hope the plugin support i18n.
- "async onClose()" i get the error from console, so i annotated it.

Comment (kkzzhizhou) [reply src/view.tsx:35]: sorry about that.
I found i did not upload the reward panel. reject this pull requests.
I will be back.

**Comment by SuperChamp234** [comment]
Okay, once your plugin is ready you can open another pull request, looking forward to it!

**Pull request closed without merging**